homeassistant/components/plugwise/climate.py

```diff
@@ -67,17 +67,17 @@ def __init__(
             self._attr_preset_modes = presets
 
         # Determine hvac modes and current hvac mode
-        self._attr_hvac_modes = [HVACMode.HEAT]
+        self._attr_hvac_modes = [HVACMode.OFF, HVACMode.HEAT]
         if self.coordinator.data.gateway["cooling_present"]:
-            self._attr_hvac_modes = [HVACMode.HEAT_COOL]
+            self._attr_hvac_modes = [HVACMode.OFF, HVACMode.HEAT_COOL]
         if self.device["available_schedules"] != ["None"]:
             self._attr_hvac_modes.append(HVACMode.AUTO)
 
         self._attr_min_temp = self.device["thermostat"]["lower_bound"]
         self._attr_max_temp = self.device["thermostat"]["upper_bound"]
-        # Ensure we don't drop below 0.1
+        # Ensure we don't drop below 0.5
         self._attr_target_temperature_step = max(
-            self.device["thermostat"]["resolution"], 0.1
+            self.device["thermostat"]["resolution"], 0.5
```

Comment:
I don't think this is a good change tbh.

Reply:
@frenck sorry for being direct, but, what is more important, a good UI-experience for the general user or your personal use?

On the other hand, if the UI could implement up/down or +/- buttons, then we could leave this as it is.

Reply:
@CoMPaTech suggests to allow the user to set the resolution, could be done via an Option. Would that be an idea?

Reply:
I was actually thinking if we could change the resolution via a service(call), as far as I see there is no 'service' for it. Not sure if we can change it as an attribute through service/yaml.

Reply:
> On the other hand, if the UI could implement up/down or +/- buttons, then we could leave this as it is.

The UI implements those. 

> sorry for being direct, but, what is more important, a good UI-experience for the general user

We should not work around UX problems in the backend.

Reply:
@frenck Ah OK the +/- buttons are there when pressing the 3 vertical dots on the climate-card.
I will remove this change.